newscoop/composer.json

```diff
@@ -42,7 +42,7 @@
         "hybridauth\/hybridauth":"dev-master",
         "smarty\/smarty":"3.1.21",
         "knplabs\/knp-menu-bundle":"2.0.*@dev",
-        "knplabs\/knp-menu":"2.0.*@dev",
+        "knplabs\/knp-menu":"2.0.1",
```

Comment:
i will update it later - so far i have some plugins installed and don't want to crash anything